src/theory/quantifiers/sygus_grammar_cons.cpp

```diff
@@ -91,14 +92,29 @@ Node CegGrammarConstructor::process( Node q, std::map< Node, Node >& templates,
     Node sfvl = sf.getAttribute(SygusSynthFunVarListAttribute());
     // sfvl may be null for constant synthesis functions
     Trace("cegqi-debug") << "...sygus var list associated with " << sf << " is " << sfvl << std::endl;
+
     TypeNode tn;
     std::stringstream ss;
     ss << sf;
     if( v.getType().isDatatype() && ((DatatypeType)v.getType().toType()).getDatatype().isSygus() ){
       tn = v.getType();
     }else{
+      // check which arguments are irrelevant
+      std::unordered_set<unsigned> arg_irrelevant;
+      std::unordered_set<Node, NodeHashFunction> term_irrelevant;
+      // convert to term
+      for (std::unordered_set<unsigned>::iterator ita = arg_irrelevant.begin();
```

Comment:
I might be missing something but it seems like this loop would never do anything since `arg_irrelevant` is always empty.